### DISCUSSION TITLE
Can SINDY be used to directly model relation as F(a,v,theta,t)?

### DISCUSSION THREAD

**Discussion by Subodh213** [dynamicslab/pysindy#545] [General]
I have experimental data of lift force and variables such as velocity, acceleration, pitch, heave, displacement . Can SINdy model direct relationship for Flift force using state variables. Most appications of Sindy find the relationship between the derivative of state variables and variables itself.
Can Sindy be used for identifying F directly? is there any reference? Thanks

**Comment by Jacob-Stevens-Haas** [comment]
`x_dot` is the left hand side of the equation.  If you pass Flift as the `x_dot=...` parameter in `SINDy.fit`, you can get it to do any regression with it's library and optimzers.
